src/content/docs/en/guides/images.mdx

```diff
@@ -91,191 +91,17 @@ import myImage from '../assets/my_image.png'; // Image is 1600x900
-  "
-  alt="A description of my image."
-  width="800"
-  height="450"
-  loading="lazy"
-  decoding="async"
-/>
-```
-
-##### widths
-
-<p><Since v="3.3.0" /></p>
-
-A list of widths to generate for the image.
-
-If provided, this value will be used to generate a `srcset` attribute on the `<img>` tag. A [`sizes` property](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/sizes) must also be provided.
-
-Do not provide a value for `densities` when using this value. Only one of these two values can be used to generate a `srcset`.
-
-Widths that are larger than the original image will be ignored to avoid upscaling the image.
-
-```astro
----
-import { Image } from 'astro:assets';
-import myImage from '../assets/my_image.png'; // Image is 1600x900
----
-<Image
-  src={myImage}
-  widths={[240, 540, 720, myImage.width]}
-  sizes={`(max-width: 360px) 240px, (max-width: 720px) 540px, (max-width: 1600px) 720px, ${myImage.width}px`}
-  alt="A description of my image."
-/>
-```
-
-```html
-<!-- Output -->
-<img
-  src="/_astro/my_image.hash.webp"
-  srcset="
-    /_astro/my_image.hash.webp 240w,
-    /_astro/my_image.hash.webp 540w,
-    /_astro/my_image.hash.webp 720w,
-		/_astro/my_image.hash.webp 1600w
-  "
-  sizes="
-    (max-width: 360px) 240px,
-    (max-width: 720px) 540px,
-    (max-width: 1600px) 720px,
-    1600px
-  "
-  alt="A description of my image."
-  width="1600"
-  height="900"
-  loading="lazy"
-  decoding="async"
-/>
-```
-
-##### format
-
-You can optionally state the [image file type](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types#common_image_file_types) output to be used.
-
-By default, the `<Image />` component will produce a `.webp` file.
-
-##### quality
-
-`quality` is an optional property that can either be:
-- a preset (`low`, `mid`, `high`, `max`) that is automatically normalized between formats.
-- a number from `0` to `100` (interpreted differently between formats).
-
-##### inferSize
-
-<p><Since v="4.4.0" /></p>
-
-Allows you to set the original `width` and `height` of a remote image automatically.
-
-By default, this value is set to `false` and you must manually specify both dimensions for your remote image.
-
-Add `inferSize` to the `<Image />` component (or `inferSize: true` to `getImage()`) to infer these values from the image content when fetched. This is helpful if you don't know the dimensions of the remote image, or if they might change:
-
-```astro mark="inferSize"
----
-import { Image } from 'astro:assets';
----
-<Image src="https://example.com/cat.png" inferSize alt="A cat sleeping in the sun." />
-```
-
-`inferSize` can fetch the dimensions of a [remote image from a domain that has not been authorized](#authorizing-remote-images), however the image itself will remain unprocessed.
-
-###### inferRemoteSize()
-
-<p><Since v="4.12.0" /></p>
+In addition to the properties above, the `<Image />` component accepts all properties accepted by the HTML `<img>` tag.
```

Comment:
This sentence happens to be duplicated in [Additional properties
](https://deploy-preview-9801--astro-docs-2.netlify.app/en/guides/images/#additional-properties). Could that be removed?